L1Trigger/TrackFindingTracklet/test/ProducerTBout.cc

```diff
@@ -0,0 +1,184 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/EDPutToken.h"
+#include "FWCore/Utilities/interface/ESGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/Common/interface/Handle.h"
+
+#include "L1Trigger/TrackTrigger/interface/Setup.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+#include "L1Trigger/TrackFindingTracklet/interface/ChannelAssignment.h"
+#include "L1Trigger/TrackFindingTracklet/interface/Settings.h"
+
+#include <string>
+#include <vector>
+#include <deque>
+#include <iterator>
+#include <cmath>
+#include <numeric>
+
+using namespace std;
+using namespace edm;
+using namespace trackerTFP;
+using namespace tt;
+
+namespace trklet {
+
+  /*! \class  trklet::ProducerTBout
```

Comment:
Do your remaining PRs results in the output streams from ProducerTBout being used as input to KFin? If so, should this be moved from test/ to plugin/?

Reply:
Yes and it will be moved with the PR introducing the change of using TBout as input of KFin.